### PR TITLE
Fax machine and Paperwork job/minigame/bounty

### PR DESCRIPTION
**Pull request by MrMelbert**
### Fax Machine

- Introducing a fresh new version of the fax machine. The fax machine sends/receives faxes and is used to process paperwork.
image

- Users can insert a paper into the fax machine and select from a list of destinations to send a copy of the paper to another fax machine. The list of destinations also includes Central Command (or the Syndicate if emagged). Sending a paper to CC / the syndicate will send a message to the admins with the fax contents, and admins can respond via fax if they'd like.
image

- Only those with bridge access (heads of staff) and lawyers can send faxes currently.

- Emagging a fax machine will reroute it from CC to The Syndicate, change its style, and remove the access requirements on it.

- Fax machines can receive papers, either from admins or from other fax machine users. 
image

- Admins have VV dropdown options for either sending just a normal text box message as a fax, or sending a formatted marked paper.
image

- Fax machines have an associated room tag and cannot be deconstructed without being unlocked with an ID.

### Paperwork

- If toggled on in the fax machine, every 5 minutes (every sseconomy tick) a random amount of paper will be sent to the machine. These papers are randomly generated paperwork that is mostly fun side lore stuff.
image
- Every paperwork has a question associated and can randomly be rolled with errors. An enterprising lawyer or bridge officer, if they're looking for side money, can print a received paperwork, answer the associated question, and either stamp it accepted (if no errors are present) or denied (if there is an error). Then, they can insert the paperwork back into a fax machine and send it for processing, where they will be paid if it is correct.

### Misc

- Corrected arguments passed to the pain vv topic, functionally identical
- Removes the old modular cryogenics DMI
- Added some more text helpers

### CODE REVIEW

File: strings/paperwork.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		"exeuction",
+		"re-education",
+		"patent",
+		"passport",
+		"paperwork"
+
+	],
+
+	"contents_court_cases": [
+		"subject is suing victim for abuse of company funds."
```
Comment (Jolly-66) [inline strings/paperwork.json:48]: "subject has been charged with fraud."
"subject is suing DonkCo for damaged against them and involved parties."
"subject is suing [REDACTED] due to sexual allegations."
"subject has a pending case against [REDACTED] and requires your signature to authorize the case."
"subject is requesting authorization for parole."
"subject wishes to repeal their latest conviction." 
"station [REDACTED] is requesting a jury, but requires authorization to send out mailers and is requesting an authorization."
"station [REDACTED]'s  is requesting a new judge after the sudden death of their previous, and is requesting an authorization."
"subject is requesting for a new lawyer and requires a sign off."
"subject is pending a parole hearing, but station [REDACTED] wishes to ignore it, requesting authorization to ignore the parole."
"subject has been caught stealing from EVA and is pending trial. Requesting if trial is permitted."
"subject has been caught stealing from R&D and is pending trial. Requesting if trial is permitted."
"subject has been caught stealing from Engineering and is pending trial. Requesting if trial is permitted."
"subject has been caught stealing from Medical and is pending trial. Requesting if trial is permitted."
"subject has been caught stealing from Security and is pending trial. Requesting if trial is permitted."
"subject has been involved in numerous bar brawls, ordered to seek medical attention, Requesting authorization to send off to a medical-ward.

File: strings/paperwork.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+	],
+
+	"contents_executions": [
+		"victim is being executed by subject for traitorous activity."
```
Comment (Jolly-66) [inline strings/paperwork.json:53]: "victim is being executed for throwing the clown out of an airlock."
"victim is being executed for bludgeoning the clown to death."
"victim is being executed for tripping the captain."
"victim is being executed for being non-human."
"victim is being executed for being human."
"victim is being executed for being a doctor."
"victim is being executed for being a scientist."
"victim is being executed for being a clown."
"victim is being executed for being a mime."
"victim is being executed for killing Ian."
"victim is being executed for killing Runtime."
"victim is being executed for mishandling of their stations Super Matter."
"victim is being executed for letting a Singularity loose."
"victim is being executed for sabotaging the Super Matter."
"victim is being executed for aiding a criminal."
"victim is being executed for no reason."
"victim is being executed for breaking out of perma."
"victim is being executed for breaking and entering."
"victim is being executed for breaking and entering of a restricted area."
"victim is being executed for shoving a security officer over."
"victim is being executed for jumping a counter"
"victims is being executed for murdering PunPun."
"victims is being executed for being different."
"victims is being executed for liking pineapples on their pizza."
"victims is being executed for breaking into the armoy."
"victims is being executed for sending ass pics to the crew."
"victims is being executed for shooting the Super Matter."

File: strings/paperwork.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+	],
+
+	"contents_patents": [
+		"subject is patenting a new technology that keeps lizardpeople warm."
```
Comment (Jolly-66) [inline strings/paperwork.json:58]: "subject is patenting a new technology that does scientists jobs for them."
"subject is patenting a new technology that replaces the Super Matter engine."
"subject is patenting a new technology that makes the HFR core engine safer."
"subject is patenting a new technology that brings back cloners."
"subject is patenting a new technology that makes weapons less effective."
"subject is patenting a new technology that rivals Donk Co."
"subject is patenting a new technology that aimed at children."
"subject is patenting a new technology that replaces the BSA."
"subject is patenting a new technology that replaces AI technology."
"subject is patenting a new technology that enhances AI technology."
"subject is patenting a new technology that enhances telecommunications technology." 
"subject is patenting a new technology that stops moths from walking into the Super Matter chamber."
"subject is patenting a new technology that stops felinids jumping on tables."
"subject is patenting a new technology that enhances surgeries."
"subject is patenting a new technology that makes bombs more explosive."
"subject is patenting a new technology that makes plasma more viable."
"subject is patenting a new technology that plasma less viable."

File: strings/paperwork.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+	],
+
+	"contents_random": [
+		"lorem ipsum."
```
Comment (Jolly-66) [inline strings/paperwork.json:63]: "Leclercs lawsuit."
"Gah-Jeelius medical bill."
"Rasha-Tanaka's requests form."
"Joey Petunias bombing charges."
"Dendy Doom's missing report."
"Aza-Zure's missing report."
"Mortimer v. Oxygen case report."
"THIS IS A TEST DOCUMENT. PLEASE IGNORE. PLEASE DO NOT SIGN. PLEASE RETURN TO YOUR STATIONS DESIGNATED OFFICER."
"Is your refrigerator running? Well, you should catch it."
"We're trying to reach you about your shuttles extended warranty."
"If you apply now, you'll be able to get the best medical coverage we have to offer! Better than NT or even Syndicate benefits! Apply today!"
We regret to inform you that your cargo shuttle parked in spot 2B has been towed. To reclaim it please contact the number below."
"PLEASE! SOMEONE! THERE'S A LIVE XENOMORPH IN MY OFFICE. AND I THINK IT'S SENDING THIS!!"
"THIS IS AN URGENT MESSAGE TO YOUR STATIONS MEDICAL STAFF. PLEASE DISPOSE OF ANY MEDICAL TOOLS AS THEY MAY HAVE BEEN CONTAMINATED WITH THE LATEST SPACE FLU VIRUS."
"Please relay this message to your stations Chief Engineer: You're required by Space Law to stop using your Super Matter engine. If you refuse, you will be fined per each mW produced. This isn't to be taken lightly. Stop using your Super Matter engine and switch to green solar energy today!"
Where do you want me to deliver the bodies? Few of them are butchered too, just let me know where the drop off point is."
"URGENT MESSAGE FOR THE CAPTAIN!! hi."
"can u help me with ym math question pls :("
"what do these buttons do???????"
"how the fuck do I order my morphine bottles on this thing??"
"whats a fax machine?"
"todays security code is:"
"todays communications decryption key is:"
"this message is from the SkyRat sector. Please give us money to buy condoms."
"this message is from the BayStation sector. We're lost can you give us directions?"
"ENCRYPTED MESSAGE."
"HAVE YOU HEARD ABOUT NANOTRASENS LATEST DISCOVERY IN PLASMA SCIENCE? ITS LEADING BREAKTHROUGH CAN CURE CANCER IN JUST MOMENTS IF YOU CONSUME MORE THAN 5 GRAMS OF PLASMA EACH DAY! PURGE THOSE TOXINS FROM YOUR BODY, CLEANSE YOUR SOUL.

Comment (CRITAWAKETS) [reply strings/paperwork.json:63]: "this message is from the Teegee Terry sector. Oook, ook, OOK! OOK! AAAAAA!"
"this message is from the Eris sector. We want our blocky blue plastic back."
"this message is from the [REDACTED] sector. [RACIAL SLUR FILTER]."
"this message is from the Goon sector. Do you want bees?"
"Eric Dawsby has proclaimed ARGH and dropped to the floor upon the discovery of his long lost brother, Deric Dawsby."

**Comment by MrMelbert** [comment]
I think it's probably good at this point, there's some optimizations I want to make to the ui data but it can be done later, and further paperwork strings can be added in other PRs

**Comment by MrMelbert** [comment]
Future idea: Have the finished paperwork be apart of cargo's bounty cube system instead of sent directly from the fax machine.

**Comment by Jolly-66** [comment]
> Future idea: Have the finished paperwork be a part of cargo's bounty cube system instead of sent directly from the fax machine. (IE, bounty to fulfill and turn in X correct paperwork)

Couldn't we have the fax machine print the cubes? Because if this is a future idea, you'd need to complete a rather large lump some of paperwork to warrant a cube, since I planned these things paying out like 1~10cr per foem faxed back
Maybe we can still leave some of it in the fax machine itself? But for only a few papers, whereas in cargo it can be a parcel of papers (preferably over 30 or something)

**Comment by Jolly-66** [comment]
Had an idea somewhere when I was at work about the fax machine printing cubes
Could just be a "special order" (non-faxable) paper that must be sent back to CC on the cargo shuttle or something

Or even the massive parcel

File: jollystation_modules/code/__HELPERS/text_helpers.dm [hunk 5]
```diff
@@ -9,3 +9,42 @@
...
 	else
 		return "[copytext_char(string, 1, len - 3)]..."
+
+/// Removes all <blah blah> tags in a string of text.
+/// May result in some formatting errors, floating whitespace. Not perfect.
+/proc/remove_all_tags(string)
+	if(!string)
+		return
+
+	var/first_carrot = findtext(string, "<")
```
Comment (prodirus) [inline jollystation_modules/code/__HELPERS/text_helpers.dm:19]: carrot

Comment (MrMelbert) [reply jollystation_modules/code/__HELPERS/text_helpers.dm]: carrot

**Review by prodirus — APPROVED** [review]

**Pull request merged**